icicle-compiler/src/Icicle/Sea/IO/Base.hs

```diff
@@ -209,8 +233,11 @@ seaOfAllocFleet states
  , "{"
  , "    ifleet_t *fleet = calloc (1, sizeof (ifleet_t));"
  , ""
- , "    fleet->max_chord_count = max_chord_count;"
- , "    fleet->chord_times     = calloc (max_chord_count, sizeof(itime_t));"
+ , "    fleet->max_chord_count    = max_chord_count;"
+ , "    fleet->chord_times        = calloc (max_chord_count, sizeof(itime_t));"
+ , "    fleet->chord_name_lengths = calloc (max_chord_count, sizeof(int64_t));"
+ , "    fleet->chord_name_offsets = calloc (max_chord_count, sizeof(int64_t));"
+ , "    fleet->chord_name_data    = (uint8_t*) calloc (chord_label_name_data_size, 1);"
```

Comment:
probably don't need to allocate these anymore (except for `chord_times`) after your fixup